src/js/modules/addTask.js

```diff
@@ -0,0 +1,24 @@
+import displayTodos from './displayTodos.js';
+
+const addTask = (formDataObj) => {
+  // get the task list from local storage
+  const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+  const description = formDataObj.get('task');
+  const completed = false;
+  const id = tasks.length + 1;
+
+  // create a new task object
+  const task = { description, completed, id };
+
+  // add the new task to the task list
+  tasks.push(task);
+
+  // save the task list to local storage
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+
+  // display the updated task list
+  displayTodos();
+};
+
```

Comment:
- [ ] Good job adding the addTask functionality. Please make sure that the task indexes are number from 1 ... such that the last added task has the index equal to the length of the array as indicated in your PR body
![pr](https://user-images.githubusercontent.com/99656687/221358411-cbfb6dd1-83df-4920-bd0b-4de83c163584.png)

 Currently, your tasks are indexed from 0 as below.
![missed](https://user-images.githubusercontent.com/99656687/221358312-2246312b-e76c-4754-b774-6820cd16bb9d.png)

Reply:
- [ ] It appears like you did not follow the instructions fully. `Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1).` Below is the index numbering when I delete an Item
![index](https://user-images.githubusercontent.com/99656687/221358754-f0fc3bd3-6479-4428-b623-83fa35b47085.png)
- [ ] Your id field is doing exactly what the index field should do. please refactor your code make the id field the index. 👍🏾

Reply:
@wayungi 

this is the screenshot from my localstorage!
![image](https://user-images.githubusercontent.com/22054464/221359180-3625e676-b86a-4218-8294-6d026533cfa4.png)

Reply:
@wayungi 

Kindly find the link to the demo video below:

[demo video](https://www.loom.com/share/9416bb2c3a654d5784fb61b0afce0533)